app/istqb-ctfl-v4-chapter3-static-testing/istqb-ctfl-v4-chapter3-static-testing.css

```
@@ -0,0 +1,542 @@
+    max-width: 100% !important;
+    background: transparent !important;
+    padding: 20px;
+    margin: 0 0 24px;
+    border-radius: var(--border-radius-lg, 12px);
+    border: 1px solid var(--color-border-tertiary, #1f2129);
+    background-color: var(--color-background-secondary, #161821) !important;
+    display: flex;
+    justify-content: center;
+}
+
+.istqb-ctfl-v4-chapter3-page .mermaid-wrapper :global(svg) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)